webhook-relay/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:20.2.0-bullseye
+
+USER root
+RUN apt-get update
+# the usual suspects and required packages
+RUN apt-get install -y aptitude magic-wormhole vim gnupg
+# get Mr. Smee on the main deck immediately
+RUN npm install --global smee-client
+
+COPY ./entry_point.sh /entry_point.sh
+RUN chmod +x /entry_point.sh
+
+# install 1PW cli client
+RUN curl -sS https://downloads.1password.com/linux/keys/1password.asc | \
+gpg --dearmor --output /usr/share/keyrings/1password-archive-keyring.gpg
+RUN echo "deb [arch=$(dpkg --print-architecture) \
+signed-by=/usr/share/keyrings/1password-archive-keyring.gpg] \
+https://downloads.1password.com/linux/debian/$(dpkg --print-architecture) \
+stable main" | \
+tee /etc/apt/sources.list.d/1password.list
+RUN mkdir -p /etc/debsig/policies/AC2D62742012EA22/
+RUN curl -sS https://downloads.1password.com/linux/debian/debsig/1password.pol | \
+tee /etc/debsig/policies/AC2D62742012EA22/1password.pol
+RUN mkdir -p /usr/share/debsig/keyrings/AC2D62742012EA22
+RUN curl -sS https://downloads.1password.com/linux/keys/1password.asc | \
+gpg --dearmor --output /usr/share/debsig/keyrings/AC2D62742012EA22/debsig.gpg
+RUN apt update && apt install 1password-cli
```

Comment:
Here's an example of installing the `op` command, so you can use it (see `entry_point.sh`) to get secrets in a shell context.